alacritty.yml

```diff
@@ -353,6 +364,9 @@
   # See `cursor.style` for available options.
   #vi_mode_style: None
 
+  # Cursor blinking interval in milliseconds.
+  #blink_interval: 750
```

Comment:
You mean force clamp it to be at least 50ms? I'm not sure if that actually provides any real benefit?

It's not like your computer suddenly explodes when you put 0 as the interval. Is this realistically something that can be messed up by accident?

Reply:
I mean I'm not sure what to expect from interval 0, from what I can see it shouldn't blink at all, however it blink, since we theoretically can't provide an interval of zero, so I was thinking to clamping it to a value that we can handle, like 50ms or something. We can handle 4-5ms though I guess.

But we can keep it as 0 since I don't think it could mess up anything.

Reply:
I suppose it might be helpful to limit it just for debugging purposes. Since a rate of `0` ms might flood the event queue and cause problems. So I think I'll just limit it to something like `10ms` to make sure other than what is visible to the user, we don't run into any internal issues.